wa/workloads/speedometer/__init__.py

```diff
@@ -48,14 +49,11 @@ class Speedometer(UiautoWorkload):
     def __init__(self, target, **kwargs):
         super(Speedometer, self).__init__(target, **kwargs)
         self.gui.timeout = 1500
-        self.gui.uiauto_params['version'] = self.version
+        self.gui.uiauto_params['version'] = self.speedometer_version
 
     def setup(self, context):
+        self.target.execute('pm clear com.android.chrome', check_exit_code=False)
```

Comment:
Now this workload is associated with the chrome application this should now be done automatically by the framework.

Reply:
Fixed in latest update. Thanks Marc